support-frontend/assets/helpers/productPrice/productOptions.js

```diff
@@ -13,6 +13,9 @@ const Sixday: 'Sixday' = 'Sixday';
 const SixdayPlus: 'SixdayPlus' = 'SixdayPlus';
 const Everyday: 'Everyday' = 'Everyday';
 const EverydayPlus: 'EverydayPlus' = 'EverydayPlus';
+const Annual: 'Annual' = 'Annual';
```

Comment:
These are not product options they are billing periods which is different. I can't see anywhere you're using them in this PR but if you need another way to do something without adding these ping me on chat and we can discuss.

Reply:
Yes, I realised these are in helpers/billingPeriods.js after you pointed that out. Thanks!